lib/index.js

```diff
@@ -49,12 +84,13 @@ export default class Pageres {
 	 * @api public
 	 */
 
-	dest(dir) {
+	dest(dir): Pageres | string { // TODO: is there any way to define retur types based on input?
```

Comment:
Same way as this https://github.com/sindresorhus/pageres/pull/274/files#r79688250